### ISSUE TITLE
GPRs for MAR09985 and MAR11307

### ISSUE THREAD

**Issue by Devlin-Moyer** [SysBioChalmers/Human-GEM#588]
### Background:
Both reactions currently have no genes associated with them:
`MAR09985: 5-aminolevulinate [c] + 2 H+ [c] ⇔ 5-aminolevulinate [e] + 2 H+ [e]`
`MAR11307: 5-aminolevulinate [c] + H+ [c] ⇔ 5-aminolevulinate [e] + H+ [e]`
According to Uniprot, [SLC15A1](https://www.uniprot.org/uniprotkb/P46059/entry) (`ENSG00000088386`) can catalyze either of these reactions, and [SLC36A1](https://www.uniprot.org/uniprotkb/Q7Z2H8/entry) (`ENSG00000123643`) can catalyze `MAR11307` but not `MAR09985`. Both of these genes are already in Human-GEM and associated with many other similar reactions.

### Proposed Changes:
- [ ] Add `ENSG00000088386` to the currently empty GPR of `MAR09985`
- [ ] Add `ENSG00000088386 or ENSG00000123643` to the currently empty GPR of `MAR11307`

**Comment by haowang-bioinfo** [comment]
based on catalytic activity of [SLC15A1](https://www.uniprot.org/uniprotkb/P46059/entry) and [SLC36A1](https://www.uniprot.org/uniprotkb/Q7Z2H8/entry), direct evidence supporting 5-aminolevulinate as substrate is lacking

**Comment by Devlin-Moyer** [comment]
Oh whoops I meant to cite [this paper](https://jpet.aspetjournals.org/content/332/1/220.short?casa_token=ew1LMkK3PL8AAAAA:aOt6JDSQHiO1WlFZKmEFWSu5Ll3pBZibrtU4N5Nvd_O9BXxqo908d8QY70XaB09ZFZ5SZ7YLTkDG) and not just Uniprot for this

**Comment by haowang-bioinfo** [comment]
> Oh whoops I meant to cite [this paper](https://jpet.aspetjournals.org/content/332/1/220.short?casa_token=ew1LMkK3PL8AAAAA:aOt6JDSQHiO1WlFZKmEFWSu5Ll3pBZibrtU4N5Nvd_O9BXxqo908d8QY70XaB09ZFZ5SZ7YLTkDG) and not just Uniprot for this

yes, this paper reports that SLC15A1 and SLC36A1 contribute to proton-coupled transport of 5-aminolevulinate in the intestine.

I wonder if the transport is in/out for a tissue, or across intestine from/to lumen (I can't access the paper's content)?

**Comment by Devlin-Moyer** [comment]
It appears to be out of the intestinal lumen into the cells lining the intestines (from the last paragraph of the intro):
>There is convincing evidence that the intestinal H+-coupled di/tripeptide transporter PepT1 (SLC15A1) can transport ALA ([Döring et al., 1998](https://jpet.aspetjournals.org/content/332/1/220.full#ref-10)) and so may be involved in ALA transport across the intestinal epithelial brush-border membrane. [...] To test this hypothesis, ALA transport by the H+-coupled amino acid transporter PAT1 (SLC36A1) ([Boll et al., 2002](https://jpet.aspetjournals.org/content/332/1/220.full#ref-7); [Chen et al., 2003](https://jpet.aspetjournals.org/content/332/1/220.full#ref-8)) was measured. PAT1 mediates uptake of GABA, other small neutral amino acids (e.g., proline, glycine, and taurine), and a large number of related amino acid analogs across the apical membrane of intestinal epithelial cells ([Thwaites et al., 1995](https://jpet.aspetjournals.org/content/332/1/220.full#ref-36), [2000](https://jpet.aspetjournals.org/content/332/1/220.full#ref-35); [Anderson et al., 2004](https://jpet.aspetjournals.org/content/332/1/220.full#ref-2); [Metzner et al., 2006](https://jpet.aspetjournals.org/content/332/1/220.full#ref-25); [Thwaites and Anderson, 2007a](https://jpet.aspetjournals.org/content/332/1/220.full#ref-33)). [...] PAT1 transport, like that of PepT1, is driven by the H+ electrochemical gradient that exists because of an area of low pH adjacent to the intestinal luminal surface, called the acid microclimate ([McEwan et al., 1988](https://jpet.aspetjournals.org/content/332/1/220.full#ref-22); [Thwaites and Anderson, 2007b](https://jpet.aspetjournals.org/content/332/1/220.full#ref-34)). By using a combination of heterologous expression in Xenopus laevis oocytes and endogenous expression in Caco-2 cell monolayers, we show that PAT1 is a novel pH-dependent, rheogenic H+/ALA transporter and that both PAT1 and PepT1 contribute to ALA uptake across the brush-border membrane of intestinal epithelial cells.

**Comment by haowang-bioinfo** [comment]
> It appears to be out of the intestinal lumen into the cells lining the intestines

then please ahead to make the change

**Comment by haowang-bioinfo** [comment]
fixed in #623
